### ISSUE TITLE
Find double parishiyos by single name

### ISSUE THREAD

**Issue by SLaks** [hebcal/hebcal-es6#467]
AFAICT, there is no way to pass `Matot` (or `Masei`) into any `Sedra` API and get the שבת of מטות־מסעי.

Is there any simple way to do that?

**Comment by mjradwin** [comment]
```js
import {Sedra} from "@hebcal/core";
const sedra = new Sedra(5785);
const hd = sedra.find(['Matot', 'Masei']); // '1 Av 5785' 
```

**Comment by SLaks** [comment]
But I only have the string `Matot`, and I don't know if it's doubled in this year 

I want to get the date that _contains_ Matot, whether as double or single.

**Comment by mjradwin** [comment]
You will need to make two calls. If you get `null` for the undoubled parsha, then look it up as a double parsha.

**Comment by SLaks** [comment]
The problem is that I'm calling it with an arbitrary user-provided parsha name.  I don't want to need to add logic in my callsite to try all possible double פשריות if it returns null.

See https://github.com/SLaks/tikkun.io/commit/cb24b75297402cfe93ad9d8cfae4f683292761f7

**Comment by mjradwin** [comment]
I don't see much of a way around trying the doubled (or undoubled) parsha if they are read together (or separately) in a given year. Somewhere, some API has to do two queries - one for the user-specified name, and the second

We don't think it is a good idea to modify `Sedra.find()` itself to do this logic, because that would break current semantics.

You'll see that we do something similar on the hebcal.com website near this block of code

https://github.com/hebcal/hebcal-web/blob/main/src/parshaCommon.js#L28

**Comment by SLaks** [comment]
Yes; that's exactly what I mean.

WDYT of adding a new API (eg, `Sedra.findContaining()`) to support this? (so that consuming projects don't need a list of double פורשיות)

I'm not sure how important my commit/use case actually is, so I'm not sure whether/when I would actually build that.

**Comment by mjradwin** [comment]
Yes, we could consider adding such a function. Keeping this open and maybe I'll get to it in the coming weeks...

**Comment by mjradwin** [comment]
Fixed in 5.6.0 - give it a whirl and let us know if it works for you!
